test/Microsoft.Health.Common.UnitTests/ExceptionExtensionsTest.cs

```diff
@@ -0,0 +1,31 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Common.Extension;
+using Xunit;
+
+namespace Microsoft.Health.Common
+{
+    public class ExceptionExtensionsTest
+    {
+        [Fact]
+        public void ExceptionNotLoggedByDefault()
+        {
+            var exception = new Exception();
+            Assert.False(exception.IsLogForwardingEnabled());
+        }
+
+        [Fact]
+        public void ExceptionSetLogForwarding()
```

Comment:
Suggestion:  It may be good to add explicit tests for the types of exceptions that you expect to be forward automatically as well.